alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/ConfigField.java

```diff
@@ -60,7 +60,7 @@
  */
 public abstract class ConfigField extends AlertSerializableModel {
     public static final String REQUIRED_FIELD_MISSING = "Required field missing";
-    public static final int MAX_FIELD_LENGTH = 511;
+    public static final int MAX_FIELD_LENGTH = 2047;
```

Comment:
We had a limit in the database of 512.  Increased to 2048 since tokens are larger.  Also made a DB change so the database doesn't care about the string length anymore.  There's no performance gain in Postgres for defining the limit like in other database implementations.  It's a matter of saving space on the disk.